src/libs/ajax.js

```diff
@@ -1338,7 +1318,7 @@ const Buckets = signal => ({
     const encodeFileName = name => encodeAnalysisName(getFileName(name))
 
     const doCopy = async (newName, newBucket, body) => {
-      fetchBuckets(
+      return fetchBuckets(
```

Comment:
I'm skeptical that `withRetryOnError` needs to be removed. The issue seems to be that the UI isn't recognizing that the copy request fails and proceeds with deleting the original. It looks like this `return` will fix that issue.

@petesantos any comment on `withRetryOnError` (added for [DC-324](https://broadworkbench.atlassian.net/browse/DC-324))?

Reply:
I'll test it again with withRetryOnError, however, I still don't believe that function is working as it should be. Let me see...

Reply:
Seems like things worked when adding back in withRetryOnError.

I think what was happening that caused me to remove it - the delete happened sneakily after the first request was triggered, then the console flooded with retries. It's likely working as intended - just the lack of this return made the outcome strange.

Reply:
Loading spinners last significantly longer with this function being used, FWIW.

Reply:
> I think what was happening that caused me to remove it - the delete happened sneakily after the first request was triggered, then the console flooded with retries. It's likely working as intended - just the lack of this return made the outcome strange.

> Loading spinners last significantly longer with this function being used, FWIW.

Makes sense. It sounds like the delete request was being sent right after the copy request, so there was possibly a race. Vs now they'd be run sequentially.